ministryofjustice/laa-review-criminal-legal-aid
# Add closed applications page

## Description of change
Add closed applications page, tab available in primary navigation
Closed application page filters to only show closed applications (status: returned)
All open applications page filters to only show open applications (status: submitted)

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-97
## Notes for reviewer
The only statuses currently are 'submitted' and 'returned' - I have mapped these to 'open' and 'closed' respectively. - I ahve left a TODO comment to update when we have more 'closed' statuses
reviewed_at is also not implemented yet, and I have put a TODO comment in to update when we are able to 
## Screenshots of changes (if applicable)

### Before changes:
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/45827968/210584829-c2270441-2495-4e0f-8848-c389667e5af4.png">

### After changes:
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/45827968/210612375-15432551-b8d7-4836-be42-2cc3110d2037.png">

## How to manually test the feature
Open the app locally, see the tabs and switch between them
Make sure that there are some applications in the local database - add them via postman if not
In the terminal
`rails c`
`ca = CrimeApplication.first`
`ca.status = "returned"`
`ca.save`

View the closed application on the Closed Applications tab


[CRIMRE-97]: https://dsdmoj.atlassian.net/browse/CRIMRE-97?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
